src/machines/java/com/enderio/machines/common/blockentity/task/SpawnerMachineTask.java

```diff
@@ -232,9 +232,13 @@ public boolean trySpawnEntity(BlockPos pos, ServerLevel level) {
                 spawned = true;
                 blockEntity.setReason(PoweredSpawnerBlockEntity.SpawnerBlockedReason.NONE);
             }
+        }
+
+        if (spawned) {
             //Clear energy after spawn
-            energyConsumed -= energyCost;
+            energyConsumed -= energyCost; //TODO same amount used for 1 or MachinesConfig.COMMON.SPAWN_AMOUNT.get() spawns?
```

Comment:
I think it'd make sense to add a config for this now :)

Reply:
Hmm, that's not what I meant with the comment. What I mean is that spawning 1 or 4 mobs now takes the same amount of power. I was wondering if we keep that or not. Changing it means we need to now the number of mobs we spawn before starting this code, as we don't know what power we will have. Personally, I think it's fine to leave it as is, as spawning multiple mobs is vanilla behavior. It also means that if you don't have the perfect spawn room, your returns will be lower.